crates/tests/agile_reference/Cargo.toml

```
@@ -0,0 +1,13 @@
+[package]
+name = "agile_reference"
```

```suggestion
name = "test_agile_reference"
```

Test crates must begin with `test_` and then rerun `cargo -p tool_yml` to regenerate the test yml script.